.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This Pr still adds the .idea directory although we want to ignore it? Could there be something wrong with `.gitignore`?

Reply:
Did you try rebasing on the master to make sure that we are not adding this file through this PR? You might have to reset HEAD locally after the rebasing to see the git doesn't track the .idea dir anymore.